FCCee/IDEA/compact/README.md

```diff
@@ -9,7 +9,7 @@ IDEA version picked from the latest version in FCCDetectors repo
 IDEA_o1_v02
 ------------
 
-Based on o1_v01 but with a detailed description of the vertex detector and the drift chamber, a place holder solenoid and the endplate absorber. Missing: SiWrapper, calorimeter, detailed muons system (coming soon).
+Based on o1_v01 but with a detailed description of the vertex detector, drift chamber and muon system, a place holder solenoid and the endplate absorber. Missing: SiWrapper, calorimeter.
 
 IDEA_o1_v03
```

Comment:
I propose to leave IDEA_o1_v02 untouched in this PR and to adapt the IDEA_o1_v03 entry in the README